biomage/experiment/download.py

```diff
@@ -128,7 +124,7 @@ def _get_samples(experiment_id, input_env):
 
 
 def _download_samples(experiment_id, input_env, output_path, use_sample_id_as_name):
-    bucket = f"{SAMPLES_BUCKET}-{input_env}"
+    bucket = f"{SAMPLES_BUCKET}-{input_env}-242905224710"
```

Comment:
Can we not hardcode this? Maybe put it as a global and and set our account ID as the default value to a get, something like `AWS_ACCOUNT_ID = env.get("AWS_ACCOUNT_ID", DEFAULT_ACCOUNT_ID")`